ICSharpCode.Decompiler/CSharp/CSharpDecompiler.cs

```diff
@@ -148,6 +149,7 @@ public static List<IILTransform> GetILTransforms()
 			new FlattenSwitchBlocks(),
 			new FixNameCollisions(),
 			new AddXmlDocumentationTransform(),
+			new ExpressionTreeConverter(),
```

Comment:
The `ExpressionTreeConverter` should run before `IntroduceQueryExpressions`, this is necessary because query expressions are sometimes compiled into expression trees.